build/build.fsx

```diff
@@ -89,6 +94,25 @@ Target "Package" (fun _ ->
             })
 )
 
+Description("Run all benchmarks. Must be run with configuration=Release.")
+Target "Benchmarks" (fun _ ->
+    if configuration <> "Release" then
+        failwith "Benchmarks can only be run in Release mode. Please re-run the build with configuration=Release."
```

Comment:
This suggestion is a bit confusing. If you run build with `build/built.bat Benchmarks configuration=Release`, it will not run as the second parameter is expected to be release mode.

Valid run:
`build\build.bat Benchmarks Release`.

I do understand that build script is runner agnostic, so probably we should rework `build.bat` and `built.sh` to not treat the second argument in a special way.

Reply:
Thanks for pointing this out. I don't tend to use those scripts (just the simple one in the project root). Do you or @alexandrnikitin have a preferred option for changing this? I'm happy to make them all work like the root `build.sh` but don't know if anyone else relies on the current behaviour.

Reply:
@dtchepak First of all I'd suggest to remove all the stub files from the `build` folder and have them in the root (`build.bat` and `build.sh`). Or vice versa. It's quite confusing to have a few entry points and remember for each time which exact one to execute.

I'd simplify build scripts to the smth like following:
```batch
"build/nuget.exe" restore "build/packages.config " -PackagesDirectory "packages"
"packages/FAKE.4.63.0/tools/Fake.exe" "build/build.fsx"
```

Also I'd move the `packages` folder to the `build/packages` folder and ignore from git - it pollutes the solution root a bit 😟

Reply:
re: build scripts, the intention was that the scripts in `build/` dir supersede the one in root. Appveyor CI uses `build.bat` already. TravisCI doesn't yet.

re: packages. I prefer to have it by default (it was in root folder but NuGet updated that some time ago and stores packages in `%userprofile%\.nuget\packages`) If possible I would use the default location, [link to the docs](https://docs.microsoft.com/en-us/nuget/consume-packages/managing-the-global-packages-and-cache-folders) Anyway I would do it in separate PR if needed.

Reply:
@alexandrnikitin It's fine to keep the `build/*` runners only. What about file content? I'd suggest to not handle arguments in a special way (1st is the build target, second - configuration), but rather simply delegate all the arguments to `fake`. Are you fine with that?

Reply:
Yes, I'm fine. JFYI, I really like [the following build/ project template](https://github.com/fsprojects/ProjectScaffold) for .NET projects.